webpack.config.js

```diff
@@ -88,6 +88,7 @@ const gutenbergPackages = [
 	'autop',
 	'blob',
 	'blocks',
+	'block-serialization-default-parser',
 	'block-serialization-spec-parser',
```

Comment:
Can we stop bundling the other one if we don't use it in Gutenberg anymore?

Reply:
a good question. I don't want to kill the PEG parser since that maintains the spec in a way no hand-written implementation can.

in my comparator PRs I'm trying to move towards a system that will automatically run the implementations against the specification in something like a CI job so that we can have our formal specification without worrying about the implementation diverging (for example, if someone makes a change to the implementation without changing the spec first)

that is, I think we want to keep the `spec-parser` wherever we need it - mainly I think we want to strip it from the default _load_ of Gutenberg but whether we _build_ it, what do you think?

Reply:
The package with transpiled code is going to be there anyway. It's really up to you and how you want to use it. If you are fine with referencing it as a regular npm package then you don't need it. If you want to consume it as part of e2e test or something which requires all Gutenberg build files then you can leave it as is. I just wanted to raise the awareness.